src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -751,7 +751,7 @@ void AuraEffect::Update(uint32 diff, Unit* caster)
             m_periodicTimer -= diff;
         else // tick also at m_periodicTimer == 0 to prevent lost last tick in case max m_duration == (max m_periodicTimer)*N
         {
-            ++m_tickNumber;
+            if (++m_tickNumber == 0) m_tickNumber = 2; // Only first go can be 1
```

Comment:
I seriously doubt a single DOT would survive longer than 4,294,967,295 ticks, however this check marries with the ones below to ensure that only the first tick is counted